src/crypto/api.ts

```diff
@@ -0,0 +1,20 @@
+import sha3 from 'js-sha3';
+import { PublicKey, Signature } from 'umbral-pre';
+
+import { fromHexString, toBytes } from '../utils';
+
+export const verifySignature = (
```

Comment:
Is it really worth it allowing union types here? I don't think `message` is ever a `string`, and the caller might as well handle the conversion (then probably this function is not needed at all).